nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -801,6 +801,13 @@ Make sure to also check the many updates in the [Nixpkgs library](#sec-release-2
 - `postgresql_11` has been removed since it'll stop receiving fixes on November
   9th 2023.
 
+
```

Comment:
seems unrelated. Did this land in this PR by accident? :eyes:

Reply:
Yes probably while rebasing